src/app/query-home/utils/query-params.ts

```diff
@@ -1,29 +1,11 @@
-export type DecodedQueryParams = {
-  isDraft?: boolean
-}
-
-export const encodeQueryParams = ({
-  isDraft = false
-}: Partial<DecodedQueryParams>) => {
+export const encodeQueryParams = () => {
   const p = new URLSearchParams()
-  encodeBool(p, isDraft, "isDraft")
   return p.toString()
 }
 
-export const decodeQueryParams = (path: string): DecodedQueryParams => {
-  const url = new URLSearchParams(path)
-  return {
-    isDraft: decodeBool(url, "isDraft")
-  }
-}
-
-const encodeBool = (params, value, name) => {
-  if (value) {
-    params.set(name, "true")
-  }
-}
-
-const decodeBool = (params, name) => {
-  const value = params.get(name)
-  return value === "true"
+// Will begin using this again when we start to encode ?version= in the url
```

Comment:
I think we should include the version in the URL. For example `/queries/:id/version/:hash`.

And the lack of a version portion simply means latest.

Reply:
I have a slight preference for the query param, but can be convinced to move it in the path. Let's discuss more when we start working on the query version history.